src/table/utils/handle-accessibility.ts

```diff
@@ -81,28 +121,43 @@ export const handleResetFocus = ({
   setFocusedCellCoord(cellCoord);
 };
 
-export const handleFocusoutEvent = (evt, shouldRefocus, keyboard) => {
-  if (keyboard.enabled && !evt.currentTarget.contains(evt.relatedTarget) && !shouldRefocus.current) {
-    evt.currentTarget.querySelector('#sn-table-announcer--01').innerHTML = '';
-    evt.currentTarget.querySelector('#sn-table-announcer--02').innerHTML = '';
+export const handleFocusoutEvent = (
+  evt: FocusEvent,
+  shouldRefocus: {
+    current: boolean;
+  },
+  keyboard: stardust.Keyboard
+) => {
+  const targetElement = evt.currentTarget as HTMLDivElement;
+  if (keyboard?.enabled && !targetElement.contains(evt.relatedTarget as Node) && !shouldRefocus.current) {
+    targetElement.querySelector('#sn-table-announcer--01')!.innerHTML = '';
+    targetElement.querySelector('#sn-table-announcer--02')!.innerHTML = '';
     // Blur the table but not focus its parent element
     // when keyboard.active is false, this has no effect
-    keyboard.blur(false);
+    keyboard.blur?.(false);
```

Comment:
blur and focusSelection need to be modified on `index.d.ts`.

Reply:
It needs to be changed in nebula. you can add a ts-ignore here. see other usages of keyboard.blur, you can copy that comment